financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundActivity.kt

```diff
@@ -93,6 +94,7 @@ class FinancialConnectionsPlaygroundActivity : AppCompatActivity() {
             hostedSurface = HOSTED_SURFACE_PAYMENT_ELEMENT,
             activityResultRegistryOwner = this,
             callback = viewModel::onCollectBankAccountForInstantDebitsLauncherResult,
+            financialConnectionsAvailability = FinancialConnectionsAvailability.Full
```

Comment:
Will we be able to toggle this in the playground?

Reply:
yes! will come soon 👏